HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -45,18 +46,23 @@ namespace {
 L1TTkMuonFilter::L1TTkMuonFilter(const edm::ParameterSet& iConfig)
     : HLTFilter(iConfig),
       l1TkMuonTag_(iConfig.getParameter<edm::InputTag>("inputTag")),
-      tkMuonToken_(consumes<l1t::TrackerMuonCollection>(l1TkMuonTag_)),
-      qualityCut_(iConfig.getParameter<edm::ParameterSet>("qualities")) {
+      tkMuonToken_(consumes<l1t::TrackerMuonCollection>(l1TkMuonTag_)) {
   min_Pt_ = iConfig.getParameter<double>("MinPt");
   min_N_ = iConfig.getParameter<int>("MinN");
   min_Eta_ = iConfig.getParameter<double>("MinEta");
   max_Eta_ = iConfig.getParameter<double>("MaxEta");
   applyQuality_ = iConfig.getParameter<bool>("applyQuality");
   applyDuplicateRemoval_ = iConfig.getParameter<bool>("applyDuplicateRemoval");
+  qualities_ = iConfig.getParameter<std::vector<int>>("qualities");
```

Comment:
`qualities_` must be sorted, otherwise `std::binary_search` will not work correctly:
```c++
std::sort(qualities_.begin(), qualities_.end());
```

Reply:
Thanks, done!